WorkforceDataApi/src/WorkforceDataApi.DevUtils/Csv/IEstablishmentsCsvDownloader.cs

```diff
@@ -0,0 +1,8 @@
+namespace WorkforceDataApi.DevUtils.Csv;
+
+public interface IEstablishmentsCsvDownloader
+{
+    string GetLatestEstablishmentsCsvFilename();
```

Comment:
Does this method need to be on the interface?

Reply:
It's used in one of the commands to get the file name + also be able to check if it's already been downloaded

Reply:
I guess I could make the actual method which does the download have an optional boolean as to whether to bother downloading if it already exists and then I could remove this method ?

Reply:
Removed the need for this method